ci/verify_bundle.py

```diff
@@ -314,6 +312,10 @@ def verify(bundle, mode="full"):
     else:
         verify_data_shape(bundle_path, net_id, config_file)
         print("data shape is verified correctly.")
+    if mode == "min":
```

Comment:
is this change for including verify_data_shape in the CPU only mode? I suspect it'll be slow and the github ci might not have enough CPU memory for all the models.